test/AssetMarket.js

```diff
@@ -48,23 +47,27 @@ describe("AssetMarket", function () {
 	describe("Sale", () => {
 		let tokenId = "0";
 		let price = "1000000000000000000";
+		let saleId = 0;
 		beforeEach(async() => {
-			const mintTx = await assetArt.safeMint(user2.address, tokenId);
+			await assetArt.safeMint(user2.address, tokenId);
 		})
 
 		it("Should fail if caller is not token owner", async () => {
-			await expect(assetMarketContract.connect(user3)["sale(address,uint256,uint256)"](assetArt.address, tokenId, price)).
+			await expect(assetMarketContract.connect(user3)["sale(address,uint256,uint256)"]
+					(assetArt.address, tokenId, price)).
 					to.be.revertedWith("You are not an asset owner");
 		})
 
 		it("Should fail if address is not approved", async () => {
-			await expect(assetMarketContract.connect(user2)["sale(address,uint256,uint256)"](assetArt.address, tokenId, price)).
-			to.be.revertedWith("not approved");
+			await expect(assetMarketContract.connect(user3)["sale(address,uint256,uint256)"]

```

Comment:
The same code as the code of the case `Should fail if caller is not token owner`